pb_tool.cfg

```diff
@@ -6,7 +6,8 @@ name: pgRoutingLayer
 # Full path to where you want your plugin directory copied. If empty,
 # the QGIS default path will be used. Don't include the plugin name in
 # the path.
-plugin_path: ~/.local/share/QGIS/QGIS3/profiles/pgrouting/python/plugins
+# plugin_path: ~/.local/share/QGIS/QGIS3/profiles/pgrouting/python/plugins/
+plugin_path: /home/vicky/.local/share/QGIS/QGIS3/profiles/pgrouting/python/plugins
```

Comment:
We can not set this parameter with this absolute path in the repository. If you have problems building code you could use pb_tool in this way (with -p you can set your plugin path):
```
pb_tool deploy -y -p ~/.local/share/QGIS/QGIS3/profiles/default/python/plugins
```